utils.js

```diff
@@ -22,6 +23,65 @@ const { Gio } = imports.gi;
 const ExtensionUtils = imports.misc.extensionUtils;
 const PanelExtension = ExtensionUtils.getCurrentExtension();
 
+const Gettext = imports.gettext.domain('eos-panel-extension');
+
+var gettext = Gettext.gettext;
+
+function getSettings() {
+    const dir = PanelExtension.dir.get_child('schemas').get_path();
+    const source = Gio.SettingsSchemaSource.new_from_directory(dir,
+        Gio.SettingsSchemaSource.get_default(), false);
+    if (!source)
+        throw new Error('Error Initializing the thingy.');
```

Comment:
The thingy?

Reply:
https://github.com/endlessm/eos-hack-extension/blob/master/utils.js#L16 - need to check better when copying stuff hehe. Fixed.

Reply:
Note that this change is  actually from https://github.com/endlessm/eos-panel-extension/pull/7 though (which this PR is on top).